db/schema.rb

```diff
@@ -87,48 +87,24 @@
 
   create_table "dimensions_items", force: :cascade do |t|
     t.string "content_id"
-    t.string "title"
-    t.string "link"
-    t.string "description"
-    t.string "organisation_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "latest"
-    t.index ["content_id", "link", "organisation_id"], name: "dimensions_items_natural_key"
+    t.string "path"
```

Comment:
In the parlance of content items we should say "base path" not "path".

For example, the content item for https://www.gov.uk/income-tax has a base path of `/income-tax`. However https://www.gov.uk/income-tax/check-youre-paying-the-right-amount originates from the same content item and also therefore also haas a base path of `/income-tax.